README.md

```diff
@@ -58,10 +59,39 @@ To [report a vulnerability or bug](https://github.com/stackrox/stackrox/security
 ---
 
 ## Deploying StackRox
-### Installation via Helm
+
+### Quick Installation via Helm
+
+StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then run the helm quick install script or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options.
+
+**<details><summary>Install StackRox via Helm Installation Script </summary>**
+
+```sh
+/bin/bash -c "$(curl --fail -fsSL https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh)"
+```
+Installing StackRox this way requires multiple CPU resources and may fail if insufficient resources are available. You may use the `--small` option in order to install StackRox on smaller clusters with limited resources.
+```sh
+/bin/bash -c "$(curl --fail -fsSL https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh)" --small
```

Comment:
The `--fail` is the long version of `-f`, so maybe let's remove the `--fail`

Reply:
Oops, done